docs/source/algorithms/Abins2D-v1.rst

```diff
@@ -0,0 +1,107 @@
+.. algorithm::
+
+Credits
+-------
+
+Author:
+    Adam Jackson
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Abins2D is a plugin for Mantid which allows scientists to generate theoretical inelastic neutron scattering spectra (INS) in 2-D :math:`(|\mathbf{q}|,\omega)` space, accounting for the limitations of real-world measurements.
+
+The intensity is calculated in the "almost-isotropic" incoherent powder-averaging approximation, as used in the 1-D Abins algorithm.
```

Comment:
lets make the 1-D Abins algorithm a link to the doc

Reply:
Do you think
```
ref:`Abins`
```

will work?

Reply:
I think you need to do it the same as in the release notes

Reply:
```
ref:`Abins <algm-Abins>`
```
does the trick in main docs. I can't tell if it works for the release not because I don't remember the incantation to rebuild the release notes cache without rebuilding all of Mantid as well :sob: